src/components/SimuladorCRA.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="bgb ph3 pa2 br4" style="display: flex; flex-direction: column; align-items: center;">
+    <h3 class="text-center">Simule seu CRA</h3>
+
+    <ul style="display: flex; flex-direction: column;">
+      <div v-for="(d, index) in this.disciplinas" v-bind:key="d.nome">
+        <li style="display: flex; flex-direction: column;">
+          <div style="display: flex; justify-content: space-between; align-items: center;">
```

Comment:
```suggestion
          <div style="flex space-between items-center;">
```

Poderia mover o resto de CSS cru pra [tailwind](https://nerdcave.com/tailwind-cheat-sheet)? Deixa o código um pouco mais confuso (além de ser uma ótima oportunidade pra aprender mais sobre)?

https://tailwindcss.com